web/keyApi.go

```diff
@@ -31,11 +32,30 @@ func (ka *keyAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		ka.createApiKeyHandler(w, r)
 	case r.Method == http.MethodDelete && keySingleApi.MatchString(r.URL.Path):
 		ka.deleteApiKeyHandler(w, r)
+	case r.Method == http.MethodGet && keyListApi.MatchString(r.URL.Path):
+		ka.listApiKeysHandler(w, r)
 	default:
 		notFoundApiError(w)
 	}
 }
 
+func (ka *keyAPI) listApiKeysHandler(w http.ResponseWriter, _ *http.Request) {
+	apiKeys, err := services.ListAPIKeys(ka.userID)
+	if err != nil {
+		slog.Error(err.Error())
+		internalServerError(w)
+		return
+	}
+	response := map[string]interface{}{"apiKeys": apiKeys}
```

Comment:
For this, can we just return the `apiKeys` data directly. Just to flatten the response a bit, seems unnecessary to nest it again.

Reply:
Also in line 66 create key, we can make a similar change